core-bundle/src/Picker/ArticlePickerProvider.php

```diff
@@ -84,8 +109,12 @@ public function getDcaAttributes(PickerConfig $config): array
     /**
      * {@inheritdoc}
      */
-    public function convertDcaValue(PickerConfig $config, $value): string
+    public function convertDcaValue(PickerConfig $config, $value)
     {
+        if ('article' === $config->getContext()) {
+            return (int) $value;
```

Comment:
I think it should return always the same type so convert this one to a string rather than to an integer. Then you could restore the return type above.

Reply:
That's just what all the other pickers to as well 🤷‍♂ 
https://github.com/contao/contao/blob/master/core-bundle/src/Picker/PagePickerProvider.php#L113